tests/dragonfly/utility.py

```diff
@@ -64,6 +65,7 @@ class ValueType(Enum):
     SET = 2
     HSET = 3
     ZSET = 4
+    JSON = 5
```

Comment:
Weird indentation

Reply:
Those are class fields - pythonic way to make an enum. Better than making it all just global flags